### PR TITLE
Update staking contract

### PR DESCRIPTION
**Pull request by netoneko**

### CODE REVIEW

File: token.js [hunk 1]
```diff
@@ -1,33 +1,54 @@
...
+    }, BigInt(0));
 }
 
 async function tokensInCirculation(web3) {
-    const contract = getContract(web3);
+    const erc20 = getERC20Contract(web3);
+    const staking = getStakingContract(web3);
+
     const block = await web3.eth.getBlockNumber();
     const updatedAt = new Date();
```
Comment (OdedWx) [inline token.js:33]: If we publish update at it's preferred to use the block timestamp

Comment (OdedWx) [reply token.js:33]: You may use:
const block = web3.eth.getBlock("latest");
const blockNumber = block.number;
const blockTime = block timestamp;

   * baseTime <- block.timestamp

**Review by OdedWx — APPROVED** [review]
Looks good, see comment on last updated

File: token.js (hunk 1 above)
Comment (netoneko) [reply token.js]: Thank you! Fixed.

**Pull request merged**